grafana-formula/grafana-formula.changes

```diff
@@ -1,3 +1,7 @@
+  * Update dashboards to use new JSON schema
+  * Fix PostgreSQL dashboard queries
```

Comment:
Does this, and the next line belong to this PR?

Reply:
Maybe yes, if you are fixing such things during the change to JSON. If so, this is OK :-)

Reply:
Thanks. Approving the PR now.